_posts/2019-07-04-tip-autoconnect-order.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: post
+title: "Automatic Connections to New Nodes"
+subtitle: "Tip of the Day"
+category: tips
+tags: [nodes]
+---
+
+<img width="200" class="float-left" style="margin-right: 20px;" src="{{ site.baseurl }}/resources/images/2019-07-04-tip-autoconnect-order/spaghetti.png" alt="Spaghetti">
+When creating _Switch_, _Group_ or _Parent_ nodes in the _Graph Editor_, Gaffer
+will automatically connect your current selection up to the new node.
+Sometimes though, you end up with something similar to the picture to the left.
+But why would it do such a thing? <br><br>
```

Comment:
Single blank line instead of two breaks.

Reply:
Same as under HTML - didn't want to divert people to 'press to read more'. Is there some way we can manually control the post excerpt?

Reply:
Just so I'm clear, you want the whole post to show up in the feed?

Reply:
Ideally - its so short, it seems like making someone go to another page its a bit long winded... what do you think?

Reply:
That doesn't really fit with the way the post index works. It's fairly standard to only include the lede, and then the rest of the content is available on click-through. There's also no reasonable way to include the gif in the index, either... do you want me to handle this one myself, since I want to add a gif? I can bang it out today.

Reply:
Sure, if you have time. I guess though, moving forward, it'd be great to find some solution where we don't have to have a click though for everything... Maybe we should have a bit of a scheme about what format/solution is best for more blog-like posts?

Reply:
I guess as another example, the way "Connect to Next" appears seems a bit odd to me with this auto-excerpt - its just a picture...